.gitignore

```diff
@@ -1,3 +1,6 @@
+# Ignore crowdInApi details for translations - contact admin if you need these
+src/constants/localisation/crowdInApi.ts
```

Comment:
This file is a requirement for the code to run locally, but it also includes an API key to crowdin - I'll share the file with @Chef-Chungus & @pancake-swap and they can circulate

Reply:
Just noticed the CI fails without this file. I'll need to find a way to include it without sharing the API key